diesel_cli/src/print_schema.rs

```diff
@@ -245,10 +245,15 @@ struct Joinable<'a>(&'a ForeignKeyConstraint);
 
 impl<'a> Display for Joinable<'a> {
     fn fmt(&self, f: &mut Formatter) -> fmt::Result {
+        let fk = if self.0.foreign_key_rust_name.is_some() {
+            self.0.foreign_key_rust_name.unwrap()
```

Comment:
This line has an error because `fmt` doesn't have ownership of `self.0` maybe.

```
error[E0507]: cannot move out of borrowed content
   --> diesel_cli/src/print_schema.rs:249:13
    |
249 |             self.0.foreign_key_rust_name.unwrap()
    |             ^^^^^^ cannot move out of borrowed content
```